etc/profile-m-z/neochat.profile

```diff
@@ -0,0 +1,75 @@
+# Firejail profile for neochat
+# Description: Matrix Client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include neochat.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/KDE/neochat
```

Comment:
Add this as blacklist to disable-programs.inc too. You'll find the section # ${HOME}/.cache directory further down in that file.